lib/utility/logger.rb

```diff
@@ -40,4 +40,24 @@ def new_line
       end
     end
   end
+
+  def self.with_logging(config, &block)
+    # Set UTC as the timezone
+    ENV['TZ'] = 'UTC'
```

Comment:
ENV['TZ'] is not logger-specific setting, I think it adds some confusion setting it here specifically, though not sure yet about the best way to share the code, can improve later

Reply:
>  not sure yet about the best way to share the code, can improve later

maybe we could rename `with_logging` to `set_execution_environment` ?

Reply:
Yep, let's move it to a different class, something like `App::Environment` and do it there!